rmf_fleet_adapter/src/rmf_fleet_adapter/TaskManager.cpp

```diff
@@ -192,6 +192,15 @@ TaskManagerPtr TaskManager::make(
         self->_handle_request(request->json_msg, request->request_id);
     });
 
+  auto reliable_transient_qos =
+    rclcpp::ServicesQoS().keep_last(10).reliable().transient_local();
```

Comment:
Since this is going to be used for logs which could experience sudden bursts that we don't want to miss, I would suggest having at least `.keep_last(100)` for logs. But since task states always send out the latest state, replacing whatever was sent before, 10 would be sufficient.

Maybe we should have a different history value for each.